DuckDuckGo/Common/Extensions/StringExtension.swift

```diff
@@ -85,6 +85,24 @@ extension String {
         self.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty
     }
 
+    var cleanStringForBookmarkSearch: String {
+        self.removeAccents()
+            .replacingOccurrences(of: "[^a-zA-Z0-9]", with: "", options: .regularExpression)
+            .lowercased()
+    }
+
+    private func removeAccents() -> String {
```

Comment:
Same as above, very generic name and no documentation.
The above naming convention comment is valid for this too.

Reply:
@federicocappelli What’s generic about the name 🤔 ? Removing accents seems clear to me, the function removes any accent from the string. What other naming suggestions do you have?

I will add some documentation so it’s clearer.

Reply:
sorry, with `generic name` I meant that it doesn't follow the usual naming convention  replacing.... or cleaning... etc